sdcm/cluster_gce.py

```diff
@@ -428,6 +428,9 @@ def _create_node_with_retries(self,
         try:
             return create_instance(**create_node_params)
         except google.api_core.exceptions.GoogleAPIError as gbe:
+            if 'already exists' in str(gbe):
+                return self._get_instances_by_name(name=name, dc_idx=dc_idx)
```

Comment:
As I understand, it is useful on the 2nd+ attempt.
If so, then shouldn't we call that `_get_instances_by_name` method first and only after nothing found call the `create_instance` function?

Reply:
I think this happens when retrying instance creation and found that instance was already created